main/src/main/java/org/mobilitydata/gtfsvalidator/validator/ShapeIncreasingDistanceValidator.java

```diff
@@ -225,4 +234,58 @@ static class EqualShapeDistanceDiffCoordinatesNotice extends ValidationNotice {
           getDistanceMeters(current.shapePtLatLon(), previous.shapePtLatLon());
     }
   }
+
+  /**
+   * Two consecutive points have equal `shape_dist_traveled` and different lat/lon coordinates in
+   * `shapes.txt` and the distance between the two points is less than 1.11m.
+   *
+   * <p>When sorted by `shape.shape_pt_sequence`, the values for `shape_dist_traveled` must increase
+   * along a shape. Two consecutive points with equal values for `shape_dist_traveled` and different
+   * coordinates indicate an error.
+   */
+  @GtfsValidationNotice(
+      severity = WARNING,
+      files = @FileRefs({GtfsShapeSchema.class, GtfsStopSchema.class}))
+  static class EqualShapeDistanceDiffCoordinatesDistanceBelowThresholdNotice
+      extends ValidationNotice {
+
+    /** The id of the faulty shape. */
+    private final String shapeId;
+
+    /** The row number from `shapes.txt`. */
+    private final int csvRowNumber;
+
+    /** The faulty record's `shape_dist_traveled` value. */
+    private final double shapeDistTraveled;
+
+    /** The faulty record's `shapes.shape_pt_sequence`. */
+    private final int shapePtSequence;
+
+    /** The row number from `shapes.txt` of the previous shape point. */
+    private final long prevCsvRowNumber;
+
+    /** The previous shape point's `shape_dist_traveled` value. */
+    private final double prevShapeDistTraveled;
+
+    /** The previous record's `shapes.shape_pt_sequence`. */
+    private final int prevShapePtSequence;
+
+    /**
+     * Actual distance traveled along the shape from the first shape point to the previous shape.
```

Comment:
@cka-y I just realized since we generate documentation automatically now that #1678 is easy to solve in this PR (fixing the typo here).

Reply:
✅ done